packages/core/src/api/exporters/html/util/simplifyBlocksRehypePlugin.ts

```diff
@@ -142,15 +168,39 @@ export function simplifyBlocks(options: SimplifyBlocksOptions) {
         // Lifts all children out of the current block, as only list items should allow nesting.
         tree.children.splice(i + 1, 0, ...blockGroup.children);
         // Replaces the block with only the content inside it.
-        tree.children[i] = blockContent.children[0];
+        const content = blockContent.children[0] as HASTElement;
```

Comment:
small code style suggestion: let's make sure we reuse the duplicated code between this part and 191+?